docs/py-client.rst

```diff
@@ -0,0 +1,47 @@
+Python client interface
+=======================
+
+The ``karabo_bridge`` Python package provides a client interface to receive data
+from the Karabo bridge.
+
+.. module:: karabo_bridge
+
+
+.. autoclass:: Client
+
+   .. automethod:: next
+
+Data container
+--------------
+
+The data object returned by :meth:`Client.next` is a nested dictionary.
+
+The first level of keys are source names, which typically correspond to Karabo
+device names. Each source has a data dictionary representing a flattened Karabo
+hash, with dots delimiting successive key levels. The values are either basic
+Python types such as strings and floats, or numpy arrays.
+
+Each source data dictionary also has a key ``metadata``,
+which contains a further nested dictionary with keys:
+``source``, ``timestamp``, ``timestamp.tid``, ``timestamp.sec`` and ``timestamp.frac``.
+
+.. code-block:: python
+
+    {
+        'SPB_DET_AGIPD1M-1/DET/detector': {
+            'image.data': array(...),
+            # ... other keys
+            'metadata': {
+                'source': 'SPB_DET_AGIPD1M-1/DET/detector',
+                'timestamp': 1526464869.4109755,
+                'timestamp.frac': '4109755',
```

Comment:
this should have attosecond resolution, so (at least comming from the karabo environment) it will look like: `'timestamp.frac': '410975500000000000',`

Reply:
I got this from the simulator - do we need to fix how that generates timestamps?